cli/src/main/java/org/apache/any23/cli/Rover.java

```diff
@@ -172,6 +174,8 @@ protected void configure() {
                                              defaultns);
         }
 
+        extractionParameters.setFlag(ExtractionParameters.EXTRACTION_WORKFLOWS_FLAG, workflow);
```

Comment:
We should not need a separate flag to enable certain extractors. If an extractor is contained within the extractor group we are using, then that should be, on its own, enough to enable itself.